### ISSUE TITLE
[BUG] CLI mode flag results in error for nxdt_host.py

### ISSUE THREAD

**Issue by PolyCatDev** [DarkMatterCore/nxdumptool#163]
## Problem
Error while trying to run nxdt_host.py in CLI mode. 
GUI mode works perfectly tho.

## Steps to reproduce
1. Set up py enviroment
2. run `python nxdt_host.py -c`

## Log
```log
nxdumptool host script v0.4. Copyright (c) 2020-2024, DarkMatterCore.
Output directory: "/var/home/polycat/Downloads/nxdumptool/host/nxdumptool".

Traceback (most recent call last):
  File "/var/home/polycat/Downloads/nxdumptool/host/nxdt_host.py", line 1430, in <module>
    ret = main()
          ^^^^^^
  File "/var/home/polycat/Downloads/nxdumptool/host/nxdt_host.py", line 1419, in main
    cliInitialize()
  File "/var/home/polycat/Downloads/nxdumptool/host/nxdt_host.py", line 1375, in cliInitialize
    usbCommandHandler()
  File "/var/home/polycat/Downloads/nxdumptool/host/nxdt_host.py", line 1057, in usbCommandHandler
    if not usbGetDeviceEndpoints():
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/home/polycat/Downloads/nxdumptool/host/nxdt_host.py", line 607, in usbGetDeviceEndpoints
    assert g_stopEvent is not None
AssertionError
```

## OS
Fedora Linux Silverblue 40

**Comment by PolyCatDev** [comment]
ok. I looked around in the code and I tracked down the issue to the `usbCommandHandler()` function. It might be Linux thing. Imma have a look at it on a Win machine when I can.

**Comment by DarkMatterCore** [comment]
Fixed in 76de7b10fda18256fa3dd9260b5869928a5c6898.
